src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,28 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class LoginPage {
+    private WebDriver driver;
+
+    private By usernameField = By.id("user-name");
+    private By passwordField = By.id("password");
+    private By loginButton = By.id("login-button");
+    private By errorMessage = By.cssSelector("[data-test=error]");
+
+    public LoginPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void login(String username, String password) {
+        driver.findElement(usernameField).sendKeys(username);
+        driver.findElement(passwordField).sendKeys(password);
+        driver.findElement(loginButton).click();
+    }
+
+    public WebElement getErrorMessage() {
+        return driver.findElement(errorMessage);
+    }
```

Comment:
should return as string 
```suggestion
    public String getErrorMessage() {
        return driver.findElement(errorMessage).getText();
    }
```

Reply:
updated